llvm/utils/repo/docker/scripts/build_install_pstore.sh

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env bash
+#===- llvm/utils/repo/docker/scripts/build_install_pstore.sh --------------===//
```

Comment:
The file should probably be called "build_install_llvm_prepo.sh" for the same reasons as above.

Reply:
I have renamed the scripts to take the 'llvm_prepo' instead of 'pstore'.